src/program/services/scrapers/shared.py

```
@@ -46,34 +47,76 @@ def _parse_results(
+                
+                rtn_instance = RTN(overridden_settings, ranking_model)
+
+            torrent: Torrent = rtn_instance.rank(
                 raw_title=raw_title,
                 infohash=infohash,
                 correct_title=correct_title,
                 remove_trash=settings_manager.settings.ranking.options[
                     "remove_all_trash"
-                ],
+                ]
+                if not manual
```

Ruff (0.14.6)

Undefined name `manual`

(F821)